docs/tutorial/tutorial-7.rst

```diff
@@ -106,45 +57,408 @@ it looks like this::
+why ``httpx`` isn't installed - it has been installed in the your *development*
+environment, but not in the packaged app.
+
+So - we need to tell Briefcase that our app has an external dependency.
+
+Updating dependencies
+=====================
+
+In the root directory of your app, there is a file named ``pyproject.toml``.
+This file contains all the app configuration details that you provided when
+you originally ran ``briefcase new``.
+
+``pyproject.toml`` is broken up into sections; one of the sections describes
+the settings for your app::
+
+    [tool.briefcase.app.hello-world]
+    formal_name = "Hello World"
+    description = "A Tutorial app"
+    icon = "src/hello_world/resources/hello-world"
+    sources = ['src/hello_world']
+    requires = []
+
+The ``requires`` option describes the dependencies of our application. It is
+a list of strings, specifying libraries (and, optionally, versions) of libraries
+that you want to be included with your app.
+
+Modify the ``requires`` setting so that it reads::
+
+    requires = [
+        "httpx",
+    ]
+
+By adding this setting, we're telling Briefcase "when you build my app, run
+``pip install httpx`` into the application bundle". Anything that would be legal
+input to ``pip install`` can be used here - so, you could specify:
+
+ * A specific library version (e.g., ``"httpx==0.19.0"``);
+ * A range of library versions (e.g., ``"httpx>=0.19"``);
+ * A path to a git repository (e.g., ``"git+https://github.com/encode/httpx"``);
+   or
+ * A local file path (However - be warned: if you give your code to someone
+   else, this path probably won't exist on their machine!)
+
+Further down in ``pyproject.toml``, you'll notice other sections that are
+operating system dependent, like ``[tool.briefcase.app.hello-world.macOS]`` and
+``[tool.briefcase.app.hello-world.windows]``. These sections *also* have a
+``requires`` setting. These settings allow you to define additional
+platform-specific dependencies - so, for example, if you need a
+platform-specific library to handle some aspect of your app, you can specify
+that library in the platform-specific ``requires`` section, and that setting
+will only be used for that platform. You will notice that the ``toga`` libraries
+are all specified in the platform-specific ``requires`` section - this is
+because the libraries needed to display a user interface are platform specific.
+
+In our case, we want ``httpx`` to be installed on all platforms, so we use the
+app-level ``requires`` setting. The app-level dependencies will always be
+installed; the platform-specific dependecies are installed *in addition* to the
+app-level ones.
+
+.. admonition:: Python only on mobile (for now!)
+
+    On desktop platforms (macOS, Windows, Linux), any ``pip``-installable can
+    be added to your requirements. On mobile platforms, your options are a
+    little more limited - you can only use *pure Python* packages i.e.,
```

Comment:
This section should be replaced with a link to [the Briefcase FAQ](https://briefcase.readthedocs.io/en/latest/background/faq.html#can-i-use-third-party-python-packages-in-my-app).